src/main/java/de/tum/in/www1/artemis/service/connectors/localci/LocalCIConnectorService.java

```diff
@@ -197,10 +197,10 @@ private void processNewPushToTestRepository(ProgrammingExercise exercise, String
             throw new LocalCIException("Could not set test cases changed flag", e);
         }
 
-        localCITriggerService.triggerBuild(solutionParticipation, commitHash);
+        localCITriggerService.triggerBuild(solutionParticipation, commitHash, true);
```

Comment:
Even though it's just a boolean it's a magic number we could get rid of - in the IDE it will be displayed anyways, so I do not have a strong opinion on that
```suggestion
        boolean isPushToTestRepository = false;
        localCITriggerService.triggerBuild(solutionParticipation, commitHash, isPushToTestRepository);
```